modules/generic_worker/templates/run-generic-worker.sh.erb

```diff
@@ -49,14 +49,20 @@ if [ $exitstatus -eq 69 ]; then
 elif [ $exitstatus -gt 69 -o $exitstatus -lt 69 ]; then
     # If exitstatus is different by 69 (grater than 69 or lower than 69)
     # The worker was removed from the quarantin and run with success a job
-    /bin/rm -rf <%= scope.lookupvar('users::builder::home') %>/quarantined
-    # Self resolve the bug
-    resolve_bug_self_failure
-    # Now reboot the machine immediately (time costs money)
-    /usr/bin/sudo /sbin/reboot
-    # Sleep to prevent this script from terminating naturally, and launchd restarting
-    # it. Instead, the shutdown should cause this script to terminate (so it won't
-    # really sleep for 2 mins). If shutdown doesn't kick in within 2 mins, it is sane
-    # for this script to exit, and allow launchd to fire up the worker again.
-    /bin/sleep 120
+    # Remove quarantined file, if file exist
+    if [ -f <%= scope.lookupvar('users::builder::home') %>/quarantined ]; then
+        /bin/rm -rf <%= scope.lookupvar('users::builder::home') %>/quarantined
+    fi
```

Comment:
The `-f` option already should only remove the file if it exists. The `-r` is only needed for recursive deletes of subdirectories - are there subdirectories that also get deleted?